<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QPushButton" name="reconnectButton">
     <property name="text">
      <string>Reconnect</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Recordlength</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="channel1VoltPDivComboBox">
         <item>
          <property name="text">
           <string>1 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.6 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 V</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="channel1ACDCComboBox">
         <item>
          <property name="text">
           <string>DC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AC</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="samplerateComboBox">
       <property name="currentIndex">
        <number>6</number>
       </property>
       <item>
        <property name="text">
         <string>48 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500 kHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200 kHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100 kHz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="channel2CheckBox">
       <property name="text">
        <string>CH2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Samplerate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="channel1CheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>CH1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="channel2VoltPDivComboBox">
         <item>
          <property name="text">
           <string>1 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.6 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 V</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="channel2ACDCComboBox">
         <item>
          <property name="text">
           <string>DC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AC</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="recordLengthSpinBox">
       <property name="suffix">
        <string> Samples</string>
       </property>
       <property name="maximum">
        <number>200000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string>Pause</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Trigger</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Channel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="triggerChannelComboBox">
        <item>
         <property name="text">
          <string>CH1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CH2</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="triggerLevelSpinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-99990000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99990000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Triggerlevel:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Single trigger</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Rising</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>rising.png</normaloff>rising.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Falling</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>falling.png</normaloff>falling.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Triggerflanke:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="enableTriggerButton">
        <property name="text">
         <string>Enable Trigger</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="smoothSpinBox">
        <property name="suffix">
         <string> V</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-10000000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Trigger-Deadzone:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
